packages/platforms/agema/x86-64/x86-64-agema-agc7648a/onlp/PKG.yml

```diff
@@ -1,5 +1,5 @@
 variables:
-  platform: x86-64-agc7648-r0
+  platform: x86-64-agc7648a-r0
```

Comment:
This does not follow the ONIE platform specification. No separate vendor-platform tokens. Is this really their identifier?